include/dev/Sim100.hpp

```diff
@@ -0,0 +1,137 @@
+#ifndef EVT_SIM100_H
+#define EVT_SIM100_H
+
+#include <cstddef>
+#include <EVT/io/types/CANMessage.hpp>
+#include <EVT/io/CAN.hpp>
+
+namespace APM::DEV {
+
+namespace IO = EVT::core::IO;
+
+/**
+ * This class represents the interface for interacting with the SIM100 ground
+ * fault detection board.  For documentation on the CAN interface please see:
+ * {@link https://sendyne.com/Products/SIM100%20Isolation%20Monitor.html}
+ */
+class Sim100 {
```

Comment:
I think the all caps style would fit here so SIM100.

Also as an aside, really like that you included the link in there, that should definitely be the standard moving forward across EVT

Reply:
Good point.  I wrote this header file before reviewing the case comments on the last PR.